### PR TITLE
make RabbitTemplate default receive queue configurable

### PR DESCRIPTION
**Pull request by ayudovin**
Make RabbitTemplate default receive queue configurable. 
this [enhancement](https://github.com/spring-projects/spring-boot/issues/11820)

### CODE REVIEW

File: spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java [hunk 1]
```diff
@@ -713,6 +713,11 @@ public void setConsumersPerQueue(Integer consumersPerQueue) {
 		 */
 		private String routingKey = "";
 
+		/**
+		 * The default queue name that will be used for synchronous receives.
```
Comment (snicoll) [inline spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java]: Configuration keys do not start with `The` or `A`. Please inspire yourself for the appendix.

**Comment by ayudovin** [comment]
org.springframework.amqp.rabbit.core.RabbitTamplate doesn't have a `getQueue` method that's why I couldn't  covering such customizations by test

**Comment by snicoll** [comment]
We use ReflectionTestUtils for that kind of things.

**Comment by ayudovin** [comment]
@snicoll, Thank you for your advise. I have added test using `ReflectionTestUtils`.

**Comment by snicoll** [comment]
@ayudovin thanks for fixing the checkstyle issue but I've already polished your PR locally and it's ready to go. Running the build before submitting the PR would probably be more effective next time, thanks!

**Comment by ayudovin** [comment]
@snicoll Sorry for that, I'll be more careful next time

**Pull request closed without merging**